rosary.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Categories=Network;WebBrowser;
+Comment=A Bible app written with Tauri
```

Comment:
this is very poor "comment". I'm sure 100% of the user won't care about the tech stack. It's irrelevant. Same apply to the appstream file.